frontend/assets/javascripts/src/modules/form/validation/existEmail.es6

```diff
@@ -0,0 +1,32 @@
+import ajax from 'ajax';
+import debounce from 'lodash/debounce'
+
+export function init() {
+    const emailField = document.getElementById('email');
+    const isUseErrorMessage = document.getElementById('form-field__error-message-email-checker');
+    const checkerURL = '/user/check-existing-email';
+
+    const validateEmail = debounce((event) => {
+        debugger;
+        let prospectiveEmail = emailField.value;
+
+        if(!prospectiveEmail.includes('@') || !prospectiveEmail.includes('.')) {
```

Comment:
@Ap0c The idea is to avoid an unnecessary request to the API. Therefore, I check that the email has at least a `@` and a `.` for the top level domain.